### PR TITLE
Adds Sign Up

### LINKED ISSUE #92: Sign Up for Frost Account
As a Po.et user, I want to be able to Sign Up with Frost so that I can register and view my works in the blockchain.

* [x] Sign Up Form
* [x] Verify Password
* [x] Register with Frost-Client
* [x] Redirect to "Dashboard"

### PR DESCRIPTION
**Pull request by jwicks31**
Resolves #92

### Product Spec
As a Po.et user, I want to be able to Sign Up with Frost so that I can register and view my works in the blockchain.

* [x]  Sign Up Form
* [x]  Verify Password
* [x]  Register with Frost-Client
* [x]  Redirect to "Dashboard"

### Description of Changes

- Adds SignUp Actions
- Updates Input Component
- Creates InputPassword Component
- Creates SignUp Form Component
- Creates SignUp Wrapper Component
- Creates SignUp Container Component
- Updates Form Component
- Adds Feature Toggles for SignUp(register)
- Adds SignUp Page Loader
- Fixes Reducers  and unit tests
- Adds SignUp Saga

### PR Review Checklist

* [ ] **Do this changes match the product spec?**
* [ ] **Are there changes in this PR that weren't originally in the product spec?**
* [ ] **Is a feature toggle introduced for this PR? Is it disabled by default?**
* [ ] **Have you tested the changes?**
* [ ] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [ ] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [ ] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [ ] **Are the added/edited files named appropriately and located correctly in the project directory structure?**

### Warning

Merging to `master` automatically triggers a deploy in production.

### CODE REVIEW

**Comment by wzalazar** [comment]
@jwicks31 I can't see the preview of Netlify.

File: src/components/atoms/InputPassword/InputPassword.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@import '../../styles/Colors';
```
Comment (wzalazar) [inline src/components/atoms/InputPassword/InputPassword.scss]: the absolute path is working in explorer web?

File: src/components/molecules/Forms/SignIn/SignIn.tsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import * as classNames from 'classnames'
-import { Input } from 'components/atoms/Input/Input'
-import { Form } from 'components/molecules/Form/Form'
 import * as React from 'react'
+
 import { Link } from 'react-router'
 
+import { Input } from '../../../atoms/Input/Input'
```
Comment (wzalazar) [inline src/components/molecules/Forms/SignIn/SignIn.tsx]: absolutes paths?

File: src/components/molecules/Forms/SignUp/SignUp.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import * as classNames from 'classnames'
+import { Input } from 'components/atoms/Input/Input'
+import { InputPassword } from 'components/atoms/InputPassword/InputPassword'
+import { Form } from 'components/molecules/Form/Form'
+import * as React from 'react'
+import { Link } from 'react-router'
+
+import './SignUp.scss'
```
Comment (wzalazar) [inline src/components/molecules/Forms/SignUp/SignUp.tsx]: can you add the empty line after import?

File: src/components/molecules/Forms/SignUp/SignUp.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    const { password, confirmPassword } = data
+
+    if (password !== confirmPassword) {
+      elements.confirmPassword.setCustomValidity(`Passwords Don't Match`)
+      return false
+    }
+
+    return true
+  }
+  readonly onChangeRepeatPassword = (e: any, data: any, elements: any): void => {
```
Comment (wzalazar) [inline src/components/molecules/Forms/SignUp/SignUp.tsx]: can you add the empty line after the declaration?

**Review by wzalazar — APPROVED** [review]

File: src/components/atoms/InputPassword/InputPassword.tsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
+import * as classNames from 'classnames'
+import * as React from 'react'
+
+import { getParsedForm } from 'helpers/helpers'
+import { Input } from '../Input/Input'
```
Comment (wzalazar) [inline src/components/atoms/InputPassword/InputPassword.tsx]: absolute path?

File: src/pages/SignUp/SignUp.layout.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+
+import { configureFeature } from '@paralleldrive/react-feature-toggles'
+import { LoadingPageContainer } from 'components/containers/LoadingPage.container'
+import { SignUpContainer } from 'components/containers/SignUp.container'
+import { LandingLayout } from '../Landing/Layout'
```
Comment (wzalazar) [inline src/pages/SignUp/SignUp.layout.tsx]: absolute path?

**Comment by jwicks31** [comment]
@wzalazar Thank you for reviewing! I've made the changes you suggested.

**Review by wzalazar — APPROVED** [review]

**Pull request merged**